src/Denner/Client/AbstractClient.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Denner\Client;
+
+use Detail\Normalization\Normalizer\Service\NormalizerAwareTrait;
+
+use GuzzleHttp\Command\Guzzle\GuzzleClient as ServiceClient;
+
+use Zend\Paginator\Adapter\ArrayAdapter;
+
+use Denner\Client\Subscriber;
+use Denner\Client\Exception;
+use Denner\Client\Collection\DefaultCollection;
+
+abstract class AbstractClient extends ServiceClient
+{
+    use NormalizerAwareTrait;
+
+    /**
+     * @param string $name
+     * @param array $arguments
+     * @return array
+     */
+    public function __call($name, array $arguments)
+    {
+        $operation = $this->getDescription()->getOperation($name);
+
+        if ($operation === null) {
+            throw new Exception\RuntimeException(
+                sprintf(
+                    'Could not find operation for: "%s"',
+                    $name
+                )
+            );
+        }
+
+        $config = $operation->toArray();
+
+        if (!isset($config['model'])) {
```

Comment:
In my opinion should not throw  an exception, but return the array result if not present.

I would also return an array if the "\Denner\Common" Namespace is not available.

Would be much more compatible, and could be used without the require-dev of denner-common (that is a "private" repository, and not available for external consumers of the library)